src/main/scala/com/ringcentral/cassandra4io/cql/package.scala

```diff
@@ -214,10 +214,15 @@ package object cql {
       ParameterizedQuery(QueryTemplate(ctx.s(args: _*), identity), HNil)
   }
 
-  implicit class CqlStringContext(ctx: StringContext) {
-    val cqlt     = new CqlTemplateStringInterpolator(ctx)
-    val cql      = new CqlStringInterpolator(ctx)
-    val cqlConst = new CqlConstInterpolator(ctx)
+  implicit class CqlStringContext(val ctx: StringContext) extends AnyVal {
+    def cqlt     = new CqlTemplateStringInterpolator(ctx)
+    def cql      = new CqlStringInterpolator(ctx)
+    def cqlConst = new CqlConstInterpolator(ctx)
+  }
+
+  implicit class UnsetOptionValueOps[A](self: Option[A]) {
+    def usingUnset(implicit aBinder: Binder[A]): BoundValue[Option[A]] =
```

Comment:
How about `.ignoreNone` / `.ignoreNull` ? Not that it really important :)

Reply:
`ignoreNone`/`ignoreNull` introduces some new naming, which is completely unrelated to an underlying mechanism (`unset`). I believe that something with `unset` will be more familiar for Cassandra users.

I also think about `withUnset`, which is fine too.